src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -118,12 +109,51 @@ function addRubricCol(questionNum) {
         $(rubricRowFragment).insertAfter(lastTd);
     }
 
+    // Add options row at the end
+    const optionsRow =
+        `<td class="align-center rubricCol-${questionNum}-${newColNumber - 1}" data-col="${newColNumber - 1}">
+            <div class="btn-group">
+                <button type="button" class="btn btn-default" id="rubric-move-col-left-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, true)" data-toggle="tooltip"
+                        data-placement="top" title="Move column left">
+                    <span class="glyphicon glyphicon-arrow-left"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionNum}-${newColNumber - 1}"
+                        onclick="removeRubricCol(${newColNumber - 1}, ${questionNum})"
+                        onmouseover="highlightRubricCol(${newColNumber - 1}, ${questionNum}, true)"
+                        onmouseout="highlightRubricCol(${newColNumber - 1}, ${questionNum}, false)">
+                    <span class="glyphicon glyphicon-remove"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubric-move-col-right-${questionNum}-${newColNumber - 1}"
+                        onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, false)" data-toggle="tooltip"
```

Comment:
`onclick` inline JS still seem to be introduced here. If the concern is with new elements not having an associated event handler attached to them, consider using delegated events instead.

Note that this is mainly for the left and right buttons; the delete button is assumed to be handled by, or will be handled by, a PR for the `onclick` events.